scripts/system/create/modules/brokenUrlReport.js

```diff
@@ -0,0 +1,390 @@
+//
+//  brokenUrlReport.js
+//
+//  Created by Alezia Kurdis on February 22, 2021.
+//  Copyright 2021 Vircadia contributors.
+//
+//  This script add reports functionalities to the Create Application.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+var brokenUrlReportHttpRequest;
```

Comment:
```suggestion
var brokenURLReportHttpRequest;
```
And so forth, throughout the file.